### DISCUSSION TITLE
Simulation on 2024 weather data

### DISCUSSION THREAD

**Discussion by ParameshwariS-tiger** [APSIMInitiative/ApsimX#9371] [Q&A]
Hi, I am using the Peanut module for the current year (2024) from January 1st to October 3rd, and I encountered the following error for one of the location during the simulation: `tav (oC) = -47.985287958843166 is outside range of -30 to 40`

Since it's not possible to download weather data up to the current date within APSIM, I used the NASA POWER API to retrieve the required weather data. 

I also tried with a different location and facing another error: `Lower bound (-100) exceeds upper bound (-999) in bounds checking: Variable is not checked`

Any assistance on resolving this issue would be greatly appreciated. Thanks!

**Comment by BrianCollinss** [comment]
Do you get the errors after downloading the data from NASA?

**Reply by ParameshwariS-tiger** [reply]
I downloaded the weather data using NASA POWER API in csv format and manually uploaded to APSIM, because APSIM couldn't provide data up to the current date (October 3rd, 2024).
After saving the file and running the simulation, I encountered two errors for different locations. I'm unsure if the issue is related to the weather data.

**Reply by BrianCollinss** [reply]
From what I see, the first error is related to the Tav in your weather file, which is too low. The second error is referring to a variable that is out of range. I think you are using -999 in lieu of missing data, which APSIM doesn't accept.

**Reply by ParameshwariS-tiger** [reply]
Got it, thanks for clarifying

**Reply by BrianCollinss** [reply]
I am not sure, but you may be able to remove Tav from the file and let APSIM replace it.

**Reply by ParameshwariS-tiger** [reply]
Okay got it, will try that thanks
